api/api/management/commands/documentmedia.py

```diff
@@ -5,12 +5,17 @@
 from django.core.management import BaseCommand
 from django.db import connection
 from django.db.models import NOT_PROVIDED
+from django.db.models.fields import Field
 
 import yaml
 
 from api.models.media import AbstractMedia
 
 
+preamble_path = Path(__file__).parents[2] / "docs" / "media_properties" / "preamble.md"
+output_path = Path(__file__).parents[3] / "media_properties.md"
```

Comment:
Nit, since these are module-level variables (they'll need to be changed below too):
```suggestion
PREAMBLE_PATH = Path(__file__).parents[2] / "docs" / "media_properties" / "preamble.md"
OUTPUT_PATH = Path(__file__).parents[3] / "media_properties.md"
```

Reply:
Okay, changed in a838395.